hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/ExternalTableIT.java

```diff
@@ -61,6 +63,45 @@ public static void cleanoutDirectory() {
         SpliceUnitTest.clearDirectory( getExternalResourceDirectory() );
     }
 
+    @Test
+    public void testNativeSparkExtractFunction() throws Exception {
+        try {
+            File tablePath = File.createTempFile("native-spark-table", ".tmp");
```

Comment:
i made some changes in https://github.com/splicemachine/spliceengine/pull/3486 to assure these temporary files are removed again. i'm waiting on this review, so if your change is in before mine i might have to adjust this as well. (this is more a note for myself to adjust this as well).
for now, **tablePath.deleteOnExit();** should be sufficient